lib/rucio/rse/protocols/protocol.py

```diff
@@ -41,16 +41,33 @@ class RSEDeterministicScopeTranslation(PolicyPackageAlgorithms):
     """
         Translates a pfn dictionary into a scope and name
     """
+
+    _algorithm_type = "pfn2lfn"
+
     def __init__(self, vo: str = 'def'):
         super().__init__()
         self.register("def", RSEDeterministicScopeTranslation._default)
         self.register("atlas", RSEDeterministicScopeTranslation._atlas)
         policy_module = vo
-        # Uses the same policy as the DeterministicTranslation
-        if super()._supports(self.__class__.__name__, policy_module):
-            self.parser = self._get_one_algorithm(self.__class__.__name__, policy_module)
-        else:
-            self.parser = self._get_one_algorithm(self.__class__.__name__, "def")
+        logger = logging.getLogger(__name__)
+        try:
+            # Use the function defined in the policy package if it's configured so
+            algo_type = self._algorithm_type
+            algorithm_name = config.config_get('policy', self._algorithm_type)
+        except (NoOptionError, NoSectionError, RuntimeError):
+            # Don't use a function from the policy package. Use one defined in this class according to vo
+            logger.debug("PFN2LFN function will not be fetched from the policy package")
+            algo_type = self.__class__.__name__
+            if super()._supports(self.__class__.__name__, policy_module):
```

Comment:
nit, for simplicity:

```suggestion
            if super()._supports(algo_type, policy_module):
```

Reply:
done